packages/react-instantsearch-hooks/src/lib/useInstantSearchApi.ts

```diff
@@ -262,3 +268,15 @@ You can ignore this warning if you are using a custom router that suits your nee
     );
   }
 }
+
+/**
+ * Gets the version of Next.js if it is available in the `window` object,
+ * otherwise it returns the version from the npm package dependencies (in SSR).
+ */
+function getNextVersion() {
+  return (
+    (typeof window !== 'undefined' &&
+      ((window as any).next?.version as string | undefined)) ||
+    process.env.npm_package_dependencies_next
```

Comment:
process.env isn't save to access either I think, would need `?.`

Reply:
Is `process` even safe ? I guess it gets transformed by most bundlers but if using UMD this fails :/